src/assets/a11y-viewer.css

```diff
@@ -222,8 +222,7 @@ main {
 }
 
 main.is-disabled {
-    pointer-events: none;
```

Comment:
@reinoute Why is this also removed? Looks unrelated to this PR. And I'd expect `pointer-events` to be `none`. As that's exactly what the feature aims to do "keyboard only" is really "no pointers".